docusaurus/src/components/ConnectorRegistry.jsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { useEffect, useState } from "react";
+
+const oss_registry_url =
+  "https://connectors.airbyte.com/api/v0/catalog/oss_catalog.json";
+const cloud_registry_url =
+  "https://connectors.airbyte.com/api/v0/catalog/cloud_catalog.json";
+const iconBase =
+  "https://raw.githubusercontent.com/airbytehq/airbyte/master/airbyte-config-oss/init-oss/src/main/resources/icons";
+const iconStyle = { maxWidth: 25 };
+const sourceBase =
+  "https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations/connectors";
+const bugsBase =
+  "https://github.com/airbytehq/airbyte/issues?q=is:open+is:issue+label"; // :connectors/source/activecampaign
+
+async function fetchCatalog(url, setter) {
+  const response = await fetch(url);
+  const registry = await response.json();
+  setter(registry);
+  return registry;
+}
+
```

Comment:
We can drop the return here
```suggestion
function fetchCatalog(url, setter) {
  const response = await fetch(url);
  const registry = await response.json();
  setter(registry);
}

```